Makefile

```
@@ -66,21 +87,9 @@ lint:
 	$(runLint)
 	$(runLint) | grep -e "\^" && exit 1 || exit 0
 
+styler:
+	$(runStyler)
```

Pareciera que `runStyler` lo usas una única vez. En este caso lo óptimo es declararlo como _phony_ ya que no hay código repetido.

```suggestion
	R -e "library(styler)" \
	  -e "style_dir('src')" \
	  -e "style_dir('tests')" \
	  -e "style_dir('R')"
```